.github/workflows/continuous-integration-workflow.yml

```diff
@@ -10,11 +10,14 @@ on:
     branches:
       - "**"
 
-jobs:  
+jobs:
   test:
     name: Test Suite
     runs-on: ubuntu-latest
     if: "! contains(toJSON(github.event.commits.*.message), '[skip-ci]')"
+    strategy:
+      matrix:
+        target: [x86_64-unknown-linux-gnu, wasm32-unknown-unknown]
```

Comment:
Is this doing what's expected here?
I'd assume both targets to be used for the runs of `cargo check` / `cargo test` below, but if so, `cargo check` for the `plonky2_evm` crate should fail because this is definitely not `no_std`.

Reply:
I was aware of the `plonky2_evm` but it was late and I wanted to run the CI first, to see if the matrix was working as expected first. It is not.